src/functionalTest/resources/features/F-002/S-021.td.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "must receive an error response for a non existing document id",
+  "_guid_": "S-021",
+  "_extends_": "F-002_Test_Data_Base",
+
+  "specs": [
+    "contains a non existing document id"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "documentId": "00000000-0000-0000-0000-000000000000"
+    }
+  },
+
+  "expectedResponse": {
+    "_extends_": "Common_404_Response",
+    "headers": {
+      "Connection" : "keep-alive",
+      "Content-Type" : "application/json",
+      "Keep-Alive" : "[[ANYTHING_PRESENT]]"
+    },
+    "body" : {
+      "timestamp" : "[[ANYTHING_PRESENT]]",
+      "status": 404,
+      "error": "Not Found",
```

Comment:
These 2 line looks redundant, as they are already inherited from Common_404_Response.

Reply:
yes. These two are inherited and removed from response body.